WordPress/src/main/res/drawable/ic_view_post_full.xml

```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="24dp"
+    android:height="24dp"
+    android:viewportWidth="24"
+    android:viewportHeight="24">
+  <path
+      android:fillColor="#FF000000"
```

Comment:
I think we might want to change the color to `@android:color/white` and rename the drawable to `ic_view_post_full_white_24dp.xml` in order to keep it consistent with other drawables.

Reply:
Also changed.

Reply:
Fixed in https://github.com/wordpress-mobile/WordPress-Android/pull/9657/commits/762018575fe953b599121659208dcf1ba5f80498